### ISSUE TITLE
How to set the xmin for the x-axis of the loss graph?

### ISSUE THREAD

**Issue by anant15** [AlexeyAB/darknet#6389]
<img width="708" alt="image" src="https://user-images.githubusercontent.com/54929615/89105971-6de76500-d443-11ea-9aea-1d3d3231d2d2.png">

I am fine-tuning yolov3 on a pre-trained on some custom dataset which is slightly different from my current data. I am just training it for 10000 iterations but the loss curve x-axis minimum value is still 0 due to which the loss curve in not properly visible as can be seen in the image above. 

Any suggestions how to change the x-minimum value?

**Comment by AlexeyAB** [comment]
use flag `-clear` at the end of training command.
and set required `max_batches=` in cfg

**Comment by anant15** [comment]
Worked, thanks a lot.
